features/cart/viewing_cart/viewing_cart_summary.feature

```diff
@@ -0,0 +1,31 @@
+@viewing_cart
+Feature: Viewing a cart summary
+    In order to see details about my order
+    As a visitor
+    I want to be able to see my cart summary
+
+    Background:
+        Given the store operates on a single channel in "France"
+
+    @ui
+    Scenario: Viewing information about empty cart
+        When I open cart summary page
+        Then I should be notified that my cart is empty
```

Comment:
But when we created scenarios for Admin then we used to "I should be notified(...)". What is more on page you see information about empty cart, not empty cart.